packages/synapse-react-client/src/components/Markdown/MarkdownSynapse.vitest.tsx

```diff
@@ -132,63 +135,75 @@ describe('MarkdownSynapse tests', () => {
     })
   })
 
-  describe('it conditonally renders no content placeholder', () => {
-    it('by default, displays empty string when wiki markdown is undefined', () => {
+  describe('it conditionally renders no content placeholder', () => {
+    it('by default, displays nothing when markdown is undefined', async () => {
       const props: MarkdownSynapseProps = {
+        ownerId: undefined,
+        wikiId: undefined,
+        objectType: undefined,
         markdown: undefined,
       }
       renderComponent(props)
 
-      const markdownField = screen.getByTestId('markdown')
-      expect(markdownField).toHaveTextContent('')
```

Comment:
Linter said that this matcher always passes -- so fix it with `toBeEmptyDOMElement`